tests/components/Header-cy.js

```diff
@@ -0,0 +1,131 @@
+describe('Header Component', function () {
```

Comment:
Could we name this "Page Header Component" instead (and then rename the file to `PageHeader-cy`)? Just to be explicit about exactly which component we're testing.